dockerfiles/Dockerfile.build

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.7.1-alpine
+
+# A container for building InfraKit
+
+RUN apk add --update git tree make
+
+ENV GOPATH=/go
+ENV PATH=/go/bin:/usr/local/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+
```

Comment:
I think you need `ENV CGO_ENABLED=0` to build static binaries too (else they are linked against Musl libc)